src/org/usfirst/frc/team2706/robot/commands/teleop/Climb.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Climb extends Command {
+
+    
+    private boolean finished = false;
+
+    public Climb() {
+    }
+
+    protected void execute(){
+            Robot.climber.Climb();
+    }
+
+    @Override
+    protected boolean isFinished() {
+        return finished;
```

Comment:
Or if you want to go the less CPU heavy way and use our ejoystick button with runWhileHeld(), set finished to false and it'll automatically cancel when released, running infinitely in one command while it is held down